selenium_webdriver/src/test/resources/features/AddItemToCart.feature

```diff
@@ -1,6 +1,10 @@
 Feature: Add item to cart
 
-  Scenario: Add item to cart
-    Given User Logged in to Main page with 'STANDARD_USER' credentials
+  Scenario Outline: Add item to cart
```

Comment:
'Scenario Outline' usage makes sense in Data-Driven test but you have only one set of values. Please add another or make a 'Scenario' instead